streampark-console/streampark-console-service/src/main/assembly/conf/config.yaml

```diff
@@ -106,6 +106,10 @@ registry:
   heartbeat-refresh-interval: 1s
   session-timeout: 3s
 
+zookeeper:
```

Comment:
Directly configuring Zookeeper will confuse users, and they will need to learn what it does. It is recommended to change to the following:
```

high-availability:
    enable: false # true
    # The list of ZooKeeper quorum peers that coordinate the high-availability
    # setup. This must be a list of the form:
    # "host1:clientPort,host2:clientPort,..." (default clientPort: 2181)
    #
    zookeeper.quorum: 192.168.100.128:2181,192.168.100.129:2181
```

Reply:
changed